.github/docker/Dockerfile.centreon-web-legacy-alma8

```diff
@@ -0,0 +1,67 @@
+# syntax=docker/dockerfile:1.3-labs
+
+FROM docker.centreon.com/proxy/almalinux:8 AS web_fresh
+
+ARG centreon_version=22.10.0-14
+
+RUN <<EOF
+
+dnf install -y dnf-plugins-core
+dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+dnf config-manager --set-enabled 'powertools'
+
+dnf module -y reset php
+dnf module -y install php:remi-8.1
+
+curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup | bash -s -- --os-type=rhel --skip-check-installed --os-version=8 --mariadb-server-version="mariadb-10.5"
+
+dnf config-manager --add-repo https://yum.centreon.com/standard/22.10/el8/stable/noarch
+dnf config-manager --add-repo https://yum.centreon.com/standard/22.10/el8/stable/x86_64
+
+dnf install --nogpgcheck -y https://yum.centreon.com/standard/22.10/el8/stable/noarch/RPMS/centreon-${centreon_version}.el8.noarch.rpm
```

Comment:
do not install centreon rpm because it will install lm, ppm, autodisco and widgets